### PR TITLE
Feat/post comment delete

### PR DESCRIPTION
**Pull request by mikco-hipe**
### OVERVIEW

Added post comment delete
The UI is not final
Uses SoftDelete to tag comments with deleted_at

### SNIPPETS
Screenshot 2023-10-13 100120
Screenshot 2023-10-13 100137
image

### CODE REVIEW

**Review by bpocrafael — APPROVED** [review]

File: laravel-app/app/Http/Controllers/CommentController.php [hunk 1]
```diff
@@ -29,4 +30,17 @@ public function store(CommentRequest $request, UserPost $post): RedirectResponse
 
         return redirect()->back()->with('success', 'Comment added successfully.');
     }
+
```
Comment (hipe01mark) [inline laravel-app/app/Http/Controllers/CommentController.php:33]: Add comment!!!

**Review by hipe01mark — APPROVED** [review]
LGTM 🙇

File: laravel-app/app/Http/Controllers/CommentController.php (hunk 1 above)
Comment (mikco-hipe) [reply laravel-app/app/Http/Controllers/CommentController.php:33]: added comment sir 🙇

**Review by hipe01mark — APPROVED** [review]
LGTM 🙇

**Review by bpocrafael — APPROVED** [review]

**Pull request merged**